### ISSUE TITLE
NullReferenceException when trying to Attach or Execute VSCode

### ISSUE THREAD

**Issue by ghost** [TheNexusAvenger/Nexus-Embedded-Editor#3]
Hey! I'm really new to using this tool and I'm struggling with attaching and executing VSCode.

The executable returned this error:
```
Serving on port 35871
        Project type: Rojo
        Project directory: C:\Users\keyco\Desktop\baseplate
NullReferenceException: Object reference not set to an instance of an object.
   at NexusEmbeddedEditor.Window.RobloxStudioWindow..ctor(BaseWindow window, ProjectStructure structure) in C:\Users\zpcoo\Desktop\EmbeddedVSCode\NexusEmbeddedEditor\NexusEmbeddedEditor\Window\RobloxStudioWindow.cs:line 50
   at NexusEmbeddedEditor.Server.ConnectRequestHandler.GetResponseData(HttpRequest request) in C:\Users\zpcoo\Desktop\EmbeddedVSCode\NexusEmbeddedEditor\NexusEmbeddedEditor\Server\ConnectRequestHandler.cs:line 56
   at Nexus.Http.Server.Http.Request.RequestHandler.GetResponse(HttpRequest request)
```

My studio has both the updated extension and Rojo connected to the localhost. My PC has both NexusEmbeddedEditor's executable open on the localhost and Rojo. 

Help is obviously appreciated 😃

**Comment by neoasync** [comment]
Oh, I should've labelled this issue as "Good First Issue" 😅

**Comment by goldenstein64** [comment]
I have this problem too! I've been browsing the code trying to debug it myself, but I can't figure out what's wrong. The same error occurs when I try to simply connect VS Code too.

Did you already fix it @neoasync?

**Comment by ghost** [comment]
No not yet, I'm pretty sure the project is dead

**Comment by TheNexusAvenger** [comment]
If this issue is still a problem, I would suggest keeping the issue open regardless of the project's activity. I do not maintain this project anymore, so a fork and pull request will be required to resolve this.

**Comment by TheNexusAvenger** [comment]
Should be fixed by goldenstein64 in Pull Request #4. Please re-open if it isn't resolved.
